src/app/[lng]/zap/[slug]/page.tsx

```diff
@@ -0,0 +1,67 @@
+import { type Metadata } from 'next';
+import { getQuestBySlug } from 'src/app/lib/getQuestBySlug';
+import { siteName } from 'src/app/lib/metadata';
+import ZapPage from 'src/app/ui/zap/ZapPage';
+import { zapMarkets } from 'src/components/Zap/zapExampleData';
+import { getSiteUrl } from 'src/const/urls';
+import type { ExtendedQuest } from 'src/types/questDetails';
+import { sliceStrToXChar } from 'src/utils/splitStringToXChar';
+
+export async function generateMetadata({
+  params,
+}: {
+  params: { slug: string };
+}): Promise<Metadata> {
+  try {
+    const { data } = await getQuestBySlug(params.slug);
+    const questData = data.data[0] || undefined;
+    if (!questData) {
+      throw new Error();
+    }
+
+    const openGraph: Metadata['openGraph'] = {
+      title: 'Jumper | Berachain',
+      description: 'Dive into the Berachain universe!',
+      siteName: siteName,
+      url: `${getSiteUrl()}/berachain/explore/${params.slug}`,
```

Comment:
Are you sure about this url? it seems to be /zap/[slug]

Reply:
@tcheee Can we align on the meta information?

Reply:
should be good by now